contracts/bonding/BondingManager.sol

```diff
@@ -522,7 +522,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
      * Returns address of elected active transcoder and its price per segment
      * @param _maxPricePerSegment Max price (in LPT base units) per segment of a stream
      */
-    function electActiveTranscoder(uint256 _maxPricePerSegment, uint256 _block, uint256 _round) external view returns (address) {
+    function electActiveTranscoder(uint256 _maxPricePerSegment, bytes32 _rand, uint256 _round) external view returns (address) {
```

Comment:
Should we rename `_rand` to `_blockHash`, and add `@param` comments for this param and `_round`?